tests/sentry/grouping/snapshots/test_variants/test_event_hash_variant/mobile@2021_02_12/group_432_event_432.pysnap

```diff
@@ -1,19 +1,124 @@
 ---
-created: '2021-07-07T15:30:52.704892Z'
+created: '2021-07-08T17:38:05.601489Z'
 creator: sentry
 source: tests/sentry/grouping/test_variants.py
 ---
 app-depth-1:
-  hash: "578743178fb72882e3c4d60b1c30d225"
-  tree_label: "abort | stripped_application_code"
+  hash: "aeed765d29d1a60cb094f66d2cd8efb2"
+  tree_label: "stripped_application_code"
   component:
     app-depth-1*
       exception*
         stacktrace*
           frame*
             function*
               "stripped_application_code"
-          frame* (marked as prefix frame by stack trace rule (category:throw +sentinel +prefix ^-group))
+        type (ignored because exception is synthetic)
+          "0x40000015 / 0x00000001"
+        value (ignored because stacktrace takes precedence)
+          "Fatal Error: <hex> / <hex>"
+--------------------------------------------------------------------------
+app-depth-2:
```

Comment:
@untitaker Are all these extra levels OK?

Reply:
yeah that's okay, this now drops into fallback mode